client/src/main/java/com/vesoft/nebula/Client.java

```diff
@@ -14,7 +14,7 @@
  */
 public interface Client extends AutoCloseable, Serializable {
 
-    public static enum NebulaCode {
```

Comment:
why  remove it?

Reply:
Enum is static and public by default

Reply:
The interface's method should be public. So we should specified it explicitly

Reply:
Methods defined in an interface is public as default. Constant values are public and static as default
[The Java™ Tutorials](https://docs.oracle.com/javase/tutorial/java/IandI/interfaceDef.html#:~:text=The%20Interface%20Body,-The%20interface%20body&text=All%20abstract%2C%20default%2C%20and%20static,public%20%2C%20static%20%2C%20and%20final%20.)